README.md

```diff
@@ -18,6 +18,11 @@ complex molecular systems for molecular dynamics simulations.
 
 To learn more, get started or contribute, check out our [website](http://mbuild.mosdef.org).
 
+### mBuild within the MoSDeF Ecosystem
+<p align="center">
+  <img src="docs/images/mosdef.svg?raw=true" alt="mBuild within the MoSDeF Ecosystem" width="1000" height="1000"/>
```

Comment:
Currently, the rendered images takes too much of the README in the view port. I think the size should be decreased.